arangod/Cluster/ClusterInfo.cpp

```diff
@@ -5087,6 +5087,11 @@ Result ClusterInfo::agencyReplan(VPackSlice const plan) {
       trxOperations.emplace_back(absl::StrCat("Plan/", subEntry),
                                  AgencyValueOperationType::SET, entry);
     }
+    if (auto entry = plan.get({"arango", "Current", subEntry});
```

Comment:
In case anyone finds the naming confusing, as a remark, `plan` is actually an entire agency dump, not a copy of `arango/Plan`.

Reply:
Yes it is